src/components/core/PivotTable.tsx

```diff
@@ -91,23 +162,45 @@ export const getGridDataSource = (
     }
 });
 
-// This is a minimal interface for AG Grid's renderer props. Run Debugger to see full object.
-// Imported styles from AG Grid dont seem compatible.
-export interface ILoadingRendererProps {
-    node: {
-        id?: any;
-    };
-    data?: any[];
-    colDef?: {
-        field: string
-    };
-}
-
-export const RowLoadingElement = (props: ILoadingRendererProps) =>
+export const RowLoadingElement = (props: ICellRendererParams) =>
     (props.node.id !== undefined
         ? <span>{props.data[props.colDef.field]}</span>
         : <LoadingComponent width={36} imageHeight={8} height={26} speed={2} />);
 
+export const getDrillIntersection = (
+    drillItems: IDrillItem[],
+    afm: AFM.IAfm
+): IDrillEventIntersectionElement[] => {
+    // Drilling needs refactoring: all '' should be replaced by null (breaking change)
+    // intersection consists of
+        // 0..1 measure
+        // 0..1 row attribute and row attribute value
+        // 0..n column attribute and column attribute values
+    return drillItems.map((drillItem: IDrillItem) => {
+        const { identifier, uri, title } = drillItem;
+        return {
+            // id: Measure localIdentifier or attribute identifier
+            // Properties default to empty strings to maintain compatibili
+            id: isDrillableItemLocalId(drillItem) ? drillItem.localIdentifier : (identifier || ''),
+            title,
+            header: {
+                uri: isDrillableItemLocalId(drillItem)
+                    ? get(
```

Comment:
I think `get(getMeasureUriOrIdentifier(afm, drillItem.localIdentifier), 'uri', uri || '')` is sufficient here

Reply:
or maybe not